packaging/distribution/pom.xml

```diff
@@ -83,25 +83,6 @@
             <plugin>
                 <artifactId>maven-dependency-plugin</artifactId>
                 <executions>
-                    <!-- Extract share-config-custom.xml -->
```

Comment:
Forget about downloading the alfresco-installer-resources jar and extracting the share-config-custom.xml from there. Just store it alongside the resources (see below).